product/management/commands/create_superuser.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+
+class Command(BaseCommand):
+    help = 'Create a superuser'
+
+    def handle(self, *args, **options):
+        username = 'levelup'
```

Comment:
do not use hardcoded credentials. Set env. variables in docker file and use values from them 
https://stackoverflow.com/questions/9554087/setting-an-environment-variable-in-virtualenv

Reply:
done